### PR TITLE
Issue 355

### LINKED ISSUE #355: Opcap OperatorInstall check when global operators installed in the cluster
## Bug Description
Opcap fails on ROSA cluster when global operators are present.

### PR DESCRIPTION
**Pull request by yashoza19**
## Description of PR
Closes #355

### CODE REVIEW

File: internal/capability/operator_cleanup.go [hunk 1]
```diff
@@ -21,12 +22,27 @@ func operatorCleanup(ctx context.Context, opts ...auditOption) auditCleanupFn {
 
 	return func(ctx context.Context) error {
 		// delete subscription
+		SubscriptionList := &operatorv1alpha1.SubscriptionList{}
```
Comment (acornett21) [inline internal/capability/operator_cleanup.go:25]: Is this meant to be exported?

File: internal/capability/operator_install.go [hunk 2]
```diff
@@ -49,8 +51,28 @@ func operatorInstall(ctx context.Context, opts ...auditOption) (auditFn, auditCl
 			return err
 		}
 
+		SubscriptionList := &operatorv1alpha1.SubscriptionList{}
```
Comment (acornett21) [inline internal/capability/operator_install.go:54]: Is this meant to be exported?

File: internal/capability/operator_cleanup.go (hunk 1 above)
Comment (acmenezes) [reply internal/capability/operator_cleanup.go:25]: You mean the upper case SubscriptionList right? @yashoza19 when you can just check if it's really needed.

Comment (acornett21) [reply internal/capability/operator_cleanup.go:25]: Yes correct, it looks like a function local variable that is being exported.

File: internal/capability/operator_install.go (hunk 2 above)
Comment (yashoza19) [reply internal/capability/operator_install.go]: Fixed. Thanks for catching this.

**Review by acornett21 — APPROVED** [review]

**Review by acmenezes — APPROVED** [review]
/lgtm

**Review by acmenezes — APPROVED** [review]
/lgtm

**Pull request merged**